app/src/main/java/edu/agh/susgame/front/gui/components/menu/components/lobby/elements/LobbyComp.kt

```diff
@@ -54,7 +64,7 @@ fun LobbyComp(
                 gameService = gameService,
                 lobbyId = lobbyInit.id,
                 name = lobbyInit.name,
-//                maxNumOfPlayers = lobbyInit.maxNumOfPlayers,
+                maxNumOfPlayers = lobbyInit.maxNumOfPlayers,
 //                gameTime = lobbyInit.gameTime
```

Comment:
To może też już usuńmy?

Chyba ma sens, żeby to serwer narzucał ten czas rozgrywki, a nie, że samemu się wybiera